WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -101,7 +101,7 @@ final class OrderListViewController: UIViewController, GhostableViewController {
 
     /// Callback closure when an order is selected
     ///
-    private var switchDetailsHandler: (OrderDetailsViewModel?) -> Void
+    private var switchDetailsHandler: ([OrderDetailsViewModel], Int) -> Void
```

Comment:
What if we add some names to the tuple parameters? Could make it easier to read and use later on, something like:

```suggestion
    private var switchDetailsHandler: (_ allViewModels: [OrderDetailsViewModel], _ index: Int) -> Void
```

Reply:
Good idea! Done in 40c1dfc8bdc3297b61b8ffbefbb6e3eea8317e94